doc/man/8/ceph-monstore-tool.rst

```diff
@@ -0,0 +1,89 @@
+:orphan:
+
+======================================================
+ ceph-monstore-tool -- ceph monstore manipulation tool
+======================================================
+
+.. program:: ceph-monstore-tool
+
+Synopsis
+========
+
+| **ceph-monstore-tool** <store path> <cmd> [args|options]
+
+
+Description
+===========
+
+:program:`ceph-monstore-tool` is similar to `ceph-kvstore-tool`. It allows
+users to manipulate MonitorDBStore's data (monmap, osdmap, etc.) offline.
+RocksDB deualt debug level is `0`. This can be changed using `--debug`.
```

Comment:
Spelling error.  Suggest: `The default RocksDB debug level ...`

Reply:
Thank you!